policy-service/src/main/java/uk/gov/gchq/palisade/service/policy/model/PolicyResponse.java

```diff
@@ -84,20 +81,56 @@ public String getResourceId() {
     }
 
     @Generated
-    public Context getContext() throws JsonProcessingException {
-        return MAPPER.treeToValue(this.context, Context.class);
+    public Context getContext()  {
+        try {
+            return MAPPER.treeToValue(this.context, Context.class);
+        } catch (JsonProcessingException e) {
+            throw new RuntimeException("Failed to get Context", e);
```

Comment:
Would recommend throwing a Serialisation error rather than the general Runtime error. Maybe something like this but there might be a better one out there
```suggestion
            throw new SerializationFailedException("Failed to get Context", e);
```

Reply:
I have looked.  The SerializationFailedException is a Spring class so in this case it is a non-starter.  I think we need to create an Exception fit for purpose.  See the ticket PAL-946 Standard coding to handle Checked Exceptions

Reply:
Happy for this to be left as a `RuntimeException` for now.